README.txt

```diff
@@ -1,21 +1,24 @@
-Methodocracy
+Methodocracy (noun):
+A cultural-political ideology which proposes that optimal decision-making, from the individual to national level, is necessarily based upon the quantity and quality of data, interpretation, and conclusions gathered through tested and reliable methodologies.
 
-"The Super-Power of Problem-Solving"
 
+Hurdles: 
+1. It is prohibitively expensive and time-consuming for the average citizen to access information due to the high cost of scientic journals/articles and the lack of a centralized database.
 
-A free website that facilitates problem-solving. Kind of like Wikipedia, but instead of logging already known information, it focuses on accumulating new knowledge using established methods, especially the scientific method. Credentialed users and average Joe's can use the site alike.
+2. Even with free and convenient access to a centralized information archive, the vast majority of the population have little training nor ability to determine the quality of data, interpretations, conclusions, and methodologies. This results in confusion and distrust for information-based-decision-making, since bad information contradicts good information and the two are indistinguishable to the untrained mind.
 
 
-We need to work together to make unlikely connections, to think more creatively as a whole than we would have alone.
```

Comment:
I didn't remove this idea. I revised it and moved it to the bottom of the doc, because it is more of a closing statement. It is a call to action, which should be done last.